forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt

```diff
@@ -27,4 +27,6 @@ A:AB$ Token | Cost$ AddCounter<0/LOYALTY> | Planeswalker$ True | TokenScript$ c_
 A:AB$ ChangeZone | Cost$ SubCounter<3/LOYALTY> | Planeswalker$ True | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | Origin$ Battlefield | Destination$ Exile | SpellDescription$ Exile target nonland permanent.
 A:AB$ PumpAll | Cost$ SubCounter<10/LOYALTY> | Planeswalker$ True | Ultimate$ True | ValidCards$ Artifact.YouCtrl,Planeswalker.YouCtrl | KW$ Indestructible | SubAbility$ DBDestroyAll | SpellDescription$ Artifacts and planeswalkers you control gain indestructible until end of turn.
 SVar:DBDestroyAll:DB$ DestroyAll | ValidCards$ Permanent.nonLand | SpellDescription$ Destroy all nonland permanents.
+DeckHas:Ability$LifeGain|Discard|Token & Type$Soldier
```

Comment:
Soldier?

Reply:
Yeah, it makes soldier tokens iirc

Reply:
It looks like that one is missing from the Oracle though, that confused me :ninja:

Reply:
🦅 👁️ I'll fix that for whoever made this card initially 😈